fetch.bs

```diff
@@ -615,7 +616,8 @@ given a <a for=/>header name</a> <var>name</var> and a string <var>type</var> fr
 <p>To
 <dfn export for="header list" id=concept-header-list-set-structured-header>set a structured field value</dfn>
 given a <a for=/>tuple</a> (<a for=/>header name</a> <var>name</var>, <a>structured field value</a>
-<var>structuredValue</var>), in a <a for=/>header list</a> <var>list</var>, run these steps:
+<var>structuredValue</var>), in a <a for=/>header list</a> <var>list</var>, run these steps. They
+return nothing.
```

Comment:
Not necessary for the setter-type algorithms IMO.

Reply:
Fair enough.